osf_tests/test_guid_annotations.py

```diff
@@ -6,29 +6,30 @@
 from osf.models import OSFUser
 from django.db.models import Max, DateTimeField
 
-from osf_tests.factories import UserFactory, PreprintFactory
+from osf_tests.factories import UserFactory, PreprintFactory, NodeFactory
 
 
 @pytest.mark.django_db
 class TestGuidAnnotations:
```

Comment:
Minor: This class and file should be renamed. Something like `TestGuidAutoInclude`.